examples/with-arc/README.md

```diff
@@ -0,0 +1,98 @@
+# Example: `with-arc`
+
+This example shows how to construct and broadcast a transaction on the Arc testnet using Turnkey.
+
+If you want to see a similar EVM demo with passkeys, head to the example [`with-ethers-and-passkeys`](../with-ethers-and-passkeys/).
```

Comment:
this example isn't super current (uses old packages); given we want to push this out ASAP, I think we should just remove mention of this until we have an updated example in place